tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -36,6 +36,48 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("ISymWrapper"),
                                 WStr("Datadog.Trace"),
                                 WStr("MSBuild")};
+
+const WSTRING general_error_message = WStr("Failed to instrument the method.");
+
+inline WSTRING GetGenericErrorMessageWithErrorCode(short errorCode)
+{
+    return general_error_message + WStr(" [Error Code: ") + shared::ToWSTRING(errorCode) + WStr("]");
+}
+
+const WSTRING invalid_probe_method_already_instrumented =
+    WStr("Dynamic Instrumentation failed to install the probe because the corresponding method is already instrumented by another product.");
+const WSTRING invalid_method_probe_probe_is_not_supported =
+    WStr("The method where the probe should have been placed is not supported for now.");
```

Comment:
```suggestion
    WStr("The method where the probe should have been placed is not supported.");
```

Reply:
Done in 7bdbd73b1bd67c0a2e90ac3c6adb3d7300cd819d.